### DISCUSSION TITLE
安卓版能不能在打包一个真单机版本

### DISCUSSION THREAD

**Discussion by occultID** [jynew/jynew#1475] [General]
能不能编译一个无需实名无需登录无需联网验证的版本
taptap版本先不说实名，每次打开都必须联网，不联网或者没网直接进不了游戏
单机单机，就是没网的时候离线也能玩才是单机嘛
拜托开发者了，国内不合规那这版本可以放github的releases嘛

**Comment by chenggongchina** [comment]
直接打开项目点安卓一键打包就可以了。

**Comment by chenggongchina** [comment]
不build的原因是到时候被一群无良商人拿去卖，现在至少还是相信github上大家自行传播的同学会自觉点。
